com/win32com/server/policy.py

```diff
@@ -210,7 +205,7 @@ def _CreateInstance_(self, clsid, reqIID):
                 win32con.HKEY_CLASSES_ROOT, regSpec % clsid
             )
         except win32api.error:
-            raise error(
+            raise COMException(
```

Comment:
I don't even know if these even make sense as COMExceptions, this is not an area I'm familiar with at all. Please advise which exception you'd like to raise.

Reply:
I think a COM exception is not quite correct - maybe `ValueError`, because ultimately this is complaining that the value of `clsid` is invalid in this context/on this machine?

I think that's true for all other exceptions in this file - could maybe argue `TypeError` for some of them, but I don't think it matters in practice - the errors below here are never expected to be hit "for real", but only while the COM object is being developed.

Reply:
Went with `ValueError` for most, and `NotImplementedError` for `_invokeex_` as the message, comment and docstring all seem to point to this fittign teh semantics of https://docs.python.org/3/library/exceptions.html#NotImplementedError